api/base/filters.py

```diff
@@ -24,20 +24,27 @@
 def sort_multiple(fields):
     fields = list(fields)
     def sort_fn(a, b):
-        while fields:
-            field = fields.pop(0)
+        sort_direction = 1
+        for index, field in enumerate(fields):
+            if field[0] == '-':
+                sort_direction = -1
+                field = field[1:]
             a_field = getattr(a, field)
             b_field = getattr(b, field)
             if a_field > b_field:
-                return 1
+                return 1 * sort_direction
             elif a_field < b_field:
-                return -1
+                return -1 * sort_direction
+            elif a_field == b_field:
+                if index < len(fields) - 1:
+                    continue
+                else:
+                    return 0
```

Comment:
I believe the last conditional here is unnecessary (our tests pass without it); let's remove it.

That is, in the case where `index == len(fields) - 1`, we will be at the end of the `for` loop, and we'll hit the immediately subsequent `return 0`. In the `<` case, the explicit `continue` doesn't have any effect because we're already at the end of the `for` block; there are no additional statements that we're skipping by explicitly `continue`ing.